app/presenters/evidence_item_index_presenter.rb

```diff
@@ -10,7 +10,7 @@ def as_json(opts = {})
         id: item.id,
         name: item.name,
         description: item.description,
-        disease: DiseasePresenter.new(item.disease),
+        disease: item.disease.nil? ? nil: DiseasePresenter.new(item.disease),
```

Comment:
I wonder if it would be cleaner to encode this logic in the disease presenter itself so you don't have to duplicate it here and in the grid presenter both (and potentially elsewhere). In the presenter you could just do something like

```
def as_json

  if disease.nil?
     nil
  else
    # existing code
   end
end
```

and then anywhere we used a disease presenter it could handle it being nil.

Reply:
I like this suggestion as a fallback but it feels weird to me to set it up in a way that the disease presenter itself _should_ have to worry about it.

Luckily with V2, this all becomes moot anyway.